mmdeploy/pytorch/functions/normalize.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='torch.nn.functional.normalize', backend='ncnn')
+def normalize__ncnn(ctx,
+                    input: torch.Tensor,
+                    p: int = 2,
+                    dim: int = 1,
+                    eps: float = 1e-12,
+                    *args,
+                    **kwargs):
+    """Rewrite `normalize` for ncnn backend.
+
+    Make sure L2 norm on channel dim and be exported to ncnn correctly.
+    """
+    if dim < 0:
+        dim += input.ndim
+    assert dim != 0, 'Should not normalize on batch index'
+    origin_func = ctx.origin_func
+    if p != 2:
+        return origin_func(input, p, dim=dim, eps=eps)
```

Comment:
The broadcast problem should be exist even when p!=2?

Reply:
Not sure. Since l2norm is common case and other form of norm is not met yet.